app/presenters/compute_resources/vmware_presenter.rb

```diff
@@ -0,0 +1,24 @@
+module ComputeResources
+  class VmwarePresenter < ::ComputeResourcePresenter
+    def vms_list_columns
+      [
+        { name: 'name', label: _('Name') },
+        { name: 'folder', label: _('Folder'), value: :path },
+        { name: 'cpus', label: _('CPUs') },
+        { name: 'memory', label: _('Memory') },
+        { name: 'operatingsystem', label: _('Operating system') },
+        # { name: 'hypervisor', label: _('Hypervisor'), value: ->(vm, _presenter){ vm.hypervisor&.name } },
```

Comment:
This is causing an IO operation for every line, as it is the only which would left, it makes a big difference if to show it or not.
@timogoebel any idea how to get the information without contacting the vSphere?